scripts/build/ruby_install.sh

```diff
@@ -1,13 +1,11 @@
 # Import the signing keys used for rvm to gpg for verification
 timer_start=$(date +%s)
 
-apt-get install ca-certificates
-
 echo "Install Ruby & required packages/gems"
 cat $BUILD_SCRIPTS_DIR/../gpg/mpapis.asc | gpg --import -
 cat $BUILD_SCRIPTS_DIR/../gpg/pkuczynski.asc | gpg --import -
 
-curl -v -sSL https://get.rvm.io | bash -s stable
+curl -v -sSL https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer | bash -s stable
```

Comment:
```suggestion
# calls to https://get.rvm.io via curl started causing CA errors, but that call would just redirect to github, 
# which uses different CA (digicert vs let's encrypt).  This will work for as long as rvm doesn't move from 
# github to another host for this file.  If that happens, try going back to rvm.io or directly to 
# w/e new thing they choose to redirect to instead
curl -v -sSL https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer | bash -s stable
```